wordplay.py

```diff
@@ -0,0 +1,244 @@
+words_no_E_A_15plus =[] 
+for word in words:
+    if len(word) >= 15:
+        if 'E' not in word and "A" not in word:
+            words_no_E_A_15plus.append(word)
+#To Display Answer:
+#print(words_no_E_A_15plus)
+        
+#Question: 
+#What are all of the words that have a B and an X and are less than 5 letters long?
+words_B_X_5less = []
+for word in words:
+    if len(word) < 5 and "B" in word and "X" in word:
+        words_B_X_5less.append(word)
+#To Display Answer:
+#print(words_B_X_5less)
+
+#Question: 
+#What are all of the words that both start and end with Y?
+words_startY_endY = []
+for word in words: 
+    if word[0] == "Y" and word[-1] == "Y":
+        words_startY_endY.append(word)
+#To Display Answer:
+#print(words_startY_endY)
+
+#Question: 
+#What are all of the words with no vowel and not even a Y?
+vowelsAndY = "AEIOUY"
+
+words_no_vowels_includingY = []
+for word in words:
+    vowel_in_word = False
+    for char in vowelsAndY:
+        if char in word:
+            vowel_in_word = True
+            break 
+    if not vowel_in_word:
+        words_no_vowels_includingY.append(word)
+# To display answer as an array
+#print(vowel_in_word)
+
+#Question: 
+# What are all of the words that have all 5 vowels, in any order?
+vowels = "AEIOU"
+words_with_all_vowels = [] 
+for word in words: 
+    vowel_in_word = True
+    for char in vowels:
+        if char not in word:
+            vowel_in_word = False
+            break
+    if vowel_in_word:
+        words_with_all_vowels.append(word)
+# To Display Answer
+# print(words_with_all_vowels)
+
+#Question: 
+# What are all of the words that have all 5 vowels, in alphabetical order? 
+words_with_alphabetical_vowels = []
+vowels = "AEIOU"
+
+for word in words:
+    word_index = 0
+    vowel_index = 0
+    while word_index < len(word) and vowel_index < len(vowels):
+        if word[word_index] == vowels[vowel_index]:
+            vowel_index += 1
+        word_index += 1
+    if vowel_index == len(vowels):
+            words_with_alphabetical_vowels.append(word)
+#to display answer:
+#print(words_with_alphabetical_vowels)
+
+
+#Question: 
+#How many words contain the substring "TYPE”?
+type_count = 0
+for word in words:
+    if "TYPE" in word:
+        type_count += 1
+# To display an answer
+# print(type_count)
+
+#Question: 
+#Create and print an array containing all of the words that end in "GHTLY"
+end_in_GHTYL = []
+for word in words:
+    if word[-5:] ==  "GHTLY":
+         end_in_GHTYL.append(word)
+#To display all the words
+#print(end_in_GHTYL)
+
+#Question:
+#What is the shortest word that contains all 5 vowels?
+
+shortest_word_all_vowels = words_with_all_vowels[0]
+for word in words_with_all_vowels:
+    if len(word) < len(shortest_word_all_vowels):
+        shortest_word_all_vowels = word
+# To display answer:
+#print(shortest_word)
+
+
+
+#What is the longest word that contains no vowels? Assumes Y is allowed. 
+
+vowels = "AEIOU"
```

Comment:
I notice that you've got a couple of repeated variables here. Consider deduplicating + moving them to the top of the file and treating them like [constants](https://realpython.com/python-constants/#user-defined-constants) to make your intentions more clear